cfn-resources/test/e2e/project/project_test.go

```diff
@@ -159,8 +159,12 @@ func getProjectIDFromStack(output *cfn.DescribeStacksOutput) string {
 	return ""
 }
 
-func deleteProjectIfExists(t *testing.T, c *LocalTestContext) {
-	_, _, err := c.atlasClient.Projects.GetOneProject(ctx.Background(), c.projectTmplObj.ProjectID)
+func cleanupResources(t *testing.T, c *LocalTestContext) {
+	err := utility.DeleteStackIfExists(t, c.cfnClient, stackName)
+	if err != nil {
+		t.Logf("Error when Deleting Stack If Exists during cleanup")
```

Comment:
log the error please
```suggestion
		t.Log("Error when Deleting Stack If Exists during cleanup", err)
```

Reply:
updated